Description: Fix the path where the python modules searches fontforge libs.
 Upstream dlopen's some libraries that have the sole extension ".so", but
 we only ship the ".so" symlinks in the -dev package---therefore, we
 change the name of the libraries to include their versions. This is kludgy,
 but shall be changed in the future.
Author: Kęstutis Biliūnas.
Forwarded: not-needed
Reviewed-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Last-Update: 2010-05-03
--- a/pyhook/loadfontforge.h
+++ b/pyhook/loadfontforge.h
@@ -11,7 +11,7 @@
 
     if ( (lib = dlopen("libgunicode" SO_EXT,RTLD_LAZY))==NULL ) {
 #ifdef PREFIX
-	lib = dlopen( PREFIX "/lib/" "libgunicode" SO_EXT,RTLD_LAZY);
+	lib = dlopen( PREFIX "/lib/" "libgunicode.so.3.0.2", RTLD_LAZY);
 #endif
     }
     if ( lib==NULL ) {
@@ -21,7 +21,7 @@
 
     if ( (lib = dlopen("libgutils" SO_EXT,RTLD_LAZY))==NULL ) {
 #ifdef PREFIX
-	lib = dlopen( PREFIX "/lib/" "libgutils" SO_EXT,RTLD_LAZY);
+	lib = dlopen( PREFIX "/lib/" "libgutils.so.1.0.3", RTLD_LAZY);
 #endif
     }
     if ( lib==NULL ) {
@@ -31,7 +31,7 @@
 
     if ( (lib = dlopen("libfontforge" SO_EXT,RTLD_LAZY))==NULL ) {
 #ifdef PREFIX
-	lib = dlopen( PREFIX "/lib/" "libfontforge" SO_EXT,RTLD_LAZY);
+	lib = dlopen( PREFIX "/lib/" "libfontforge.so.1.0.0", RTLD_LAZY);
 #endif
     }
     if ( lib==NULL ) {
